internal/generator/vector/input/viaq_application_with_infra_includes_excludes.toml

```diff
@@ -0,0 +1,78 @@
+# Logs from containers (including openshift containers)
+[sources.input_my_app_container]
+type = "kubernetes_logs"
+max_read_bytes = 3145728
+glob_minimum_cooldown_ms = 15000
+auto_partial_merge = true
+include_paths_glob_patterns = ["/var/log/pods/test-ns-foo_*/*/*.log", "/var/log/pods/openshift-logging_*/*/*.log", "/var/log/pods/kube-apiserver_*/*/*.log"]
+exclude_paths_glob_patterns = ["/var/log/pods/default_*/*/*.log", "/var/log/pods/test-ns1_*/*/*.log", "/var/log/pods/*/*/*.gz", "/var/log/pods/*/*/*.tmp"]
```

Comment:
You mentioned to me at one point about the 'gz' and 'tmp' file here.  Given we explicitly say gather 'log' files I wonder if there is any need to actually add these excludes?  I can see deferring any change until after this PR lands

Reply:
In my understanding, if `include_paths_glob_patterns` is defined, we don't really need to define any other excludes. 
In this specific `TOML`, it will include the 3 logs defined and then it will exclude from that include list if that makes sense.

From the [Vector docs](https://vector.dev/docs/reference/configuration/sources/file/#exclude)
> [excludes] Takes precedence over the include option. Note: The exclude patterns are applied after the attempt to glob everything in include. This means that all files are first matched by include and then filtered by the exclude patterns. This can be impactful if include contains directories with contents that are not accessible.

It doesn't change anything here if we keep them in.